api/management/swagger-api_management.yaml

```diff
@@ -1107,6 +1107,10 @@ components:
           type: boolean
         show_account_deletion_button:
           type: boolean
+        cancel_registration_url:
+          type: string
+        confirmed_registration_url:
```

Comment:
Does this field need to be renamed according to @bsoniam comments?

Reply:
Yes, I did it